web/src/app/modules/sugarloaf/components/smart/navigation/navigation.component.html

```diff
@@ -1,58 +1,106 @@
+          </ng-container>
+        </ng-container>
+
+        <ng-template #noChildCollapsed>
+              <span class="tooltip-override tooltip tooltip-md">
+                <a clrVerticalNavLink
+                   [ngClass]= "{'item-selected' : i == this.lastSelection}"
+                   [routerLinkActiveOptions]="{exact: true}"
+                   [routerLink]="formatPath(section.path)"
+                   routerLinkActive="active"
+                   (mouseover)="flyoutIndex= -1; lastSelection= i"
+                   (click)="closePopups(i)"
+                >
+                    <clr-icon [attr.shape]="section.iconName | default:'home'" clrVerticalNavIcon></clr-icon>
+                </a>
+                <span class="tooltip-content-override tooltip-content">
+                  <span>{{ section.title }}</span>
+                </span>
+              </span>
+        </ng-template>
+
+      </ng-container>
+      <!-- Expanded Navigation -->
+      <ng-template #expanded>
+        <ng-template [ngIf]="section.module">
+          <div class="nav-divider"></div>
+          <span class="title">{{ section.module }}</span>
+        </ng-template>
+        <ng-container *ngIf="section.children?.length > 0; else noChildExpanded">
+          <clr-vertical-nav-group
+            routerLinkActive="active"
+            [clrVerticalNavGroupExpanded]="shouldExpand(i)"
+            (clrVerticalNavGroupExpandedChange)="setNavState($event, i)"
+          >
+            <clr-icon [attr.shape]="section.iconName | default:'home'" clrVerticalNavIcon></clr-icon>
+          {{ section.title }}
+            <clr-vertical-nav-group-children>
+              <a
+                clrVerticalNavLink
+                *ngFor="let category of section.children; trackBy: identifyNavigationItem"
+                [routerLink]="formatPath(category.path)"
+                [routerLinkActiveOptions]="{exact: true}"
+                routerLinkActive="active"
+              >
+                <clr-icon *ngIf="itemIcon(category) as categoryIcon" [attr.shape]="categoryIcon" clrVerticalNavIcon></clr-icon>
+              {{ category.title }}
+              </a>
+            </clr-vertical-nav-group-children>
+          </clr-vertical-nav-group>
+        </ng-container>
+        <ng-template #noChildExpanded>
+          <a clrVerticalNavLink
+             (click)="closePopups(i)"
+             [routerLinkActiveOptions]="{exact: true}"
+             [routerLink]="formatPath(section.path)"
+             routerLinkActive="active"
+          >
+            <clr-icon [attr.shape]="section.iconName | default:'home'" clrVerticalNavIcon></clr-icon>
+            <div class="nav-item-text">
+            {{ section.title }}
+            </div>
+          </a>
+        </ng-template>
+      </ng-template>
+    </ng-container>
+  </div>
+
+  <!-- Theme switcher -->
+  <div class="theme-switcher">
+    <app-theme-switch-button [collapsed]="collapsed"></app-theme-switch-button>
```

Comment:
nit: The theme switch button in a collapsed navigation should remain on the bottom rather than shifting upwards as a result of collapsed content.

Reply:
This was bugging me too, finally figured out how to do it